testsuite/tests/typing-modes/currying.ml

```diff
@@ -87,33 +87,35 @@ let app1 (f : a:int -> b:local_ int ref -> unit -> unit) = f ~b:(local_ ref 42)
 Line 1, characters 64-79:
 1 | let app1 (f : a:int -> b:local_ int ref -> unit -> unit) = f ~b:(local_ ref 42) ()
                                                                     ^^^^^^^^^^^^^^^
-Error: This value escapes its region.
-  Hint: It is captured by a partial application.
+Error: This value is "local" but expected to be "global"
+       because it is captured by a partial application which is "global"
+       because it is a function return value without an exclave annotation.
```

Comment:
This "it is" might be understood as refering to the underlined code, which is not right.

Maybe have a new line for "captured by a partial application" to make it more obvious what "it is" is refering to.